src/Renderer/LayeredMaterial.js

```diff
@@ -277,23 +280,20 @@ LayeredMaterial.prototype.getLayerTextureOffset = function(layer) {
     }
 };
 
-LayeredMaterial.prototype.setTexturesLayer = function(textures, layer) {
-
+LayeredMaterial.prototype.setTexturesLayer = function(textures, layer, slotOffset) {
     for (var i = 0, max = textures.length; i < max; i++) {
-
-        if (textures[i])
-            this.setTexture(textures[i].texture, layer, i, textures[i].pitch);
-
+        if (textures[i]) {
+            this.setTexture(textures[i].texture, layer, i + (slotOffset || 0), textures[i].pitch);
+        }
     }
 };
 
-LayeredMaterial.prototype.getDelta = function() {
-    if (this.paramLayers[0])
-        // TODO: Fix Me, this function always used 1st layer
-        return 0; // this.paramLayers[0].y;
-    else
+LayeredMaterial.prototype.getDelta = function(layerIndex) {
```

Comment:
it is a bad thing
originally it's delta level between _**WGS84G layer**_ and _**PM layer**_. 
I had chosen : `PMLayer.level === tile.level + 1`